presto-docs/src/main/sphinx/security/file-based-system-access-control.rst

```diff
@@ -0,0 +1,93 @@
+================================
+File Based System Access Control
+================================
+
+Presto offers a file based ``SystemAccessControl`` plugin that supports
+specifying access control rules using JSON in a file. In order to enable the
+plugin, you must add an ``etc/access-control.properties`` file containing two
+required properties, ``access-control.name`` which must be equal to ``file``,
+and ``security.config-file`` which must be set to the location of the JSON
+file. For example, if a JSON file named ``rules.json`` resides in ``etc``,
+add an ``etc/access-control.properties`` with the following contents:
+
+.. code-block:: none
+
+   access-control.name=file
+   security.config-file=etc/rules.json
+
+The ``rules.json`` file should consist of a list of access control rules.
+These rules should be specified in the order in which they are intended to be
+matched. The user is granted the privileges from the first matching rule. All
+regular expressions default to ``.*`` if not specified.
+
+At present, only the catalog access control rules are supported in this
```

Comment:
suggest:
Currently, only catalog access control rules are supported in this plugin. If you want to limit access on a system level in any other way, you must implement a custom ``SystemAccessControl`` plugin (see https://prestodb.io/docs/current/develop/system-access-control.html).